### PR TITLE
BVT consolidation - part 3

### PR DESCRIPTION
**Pull request by xian123**
1. Merge the below test cases into one case named VERIFY-LINUX-CONFIGURATION.
    BVT-IS-ROOT-PASSWORD-DELETED
    BVT-VERIFY-SSHD-CONFIG
    BVT-LAST-CONSOLE-CHECK
    BVT-VERIFY-BASH-HISTORY

2. Merge the below test cases into one case named VERIFY-LINUX-DISK-SETUP.
    BVT-ROOT-DEVICE-TIMEOUT-CHECK
    BVT-SWAP-CHECK
    BVT-MTAB-ENTRY-CHECK
    BVT-VERIFY-UUID-FSTAB

### CODE REVIEW

**Comment by xian123** [comment]
```
Test results:
[LISAv2 Test Results Summary]
Test Run On           : 04/18/2019 05:34:49
ARM Image Under Test  : canonical : ubuntuserver : 18.04-lts : Latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:11

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
------------------------------------------------------------------------------------------------------------------
    1 CORE                 VERIFY-LINUX-CONFIGURATION                                     PASS                 3.26


[LISAv2 Test Results Summary]
Test Run On           : 04/18/2019 05:34:59
ARM Image Under Test  : canonical : ubuntuserver : 18.04-lts : Latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:12

   ID TestArea             TestCaseName                                                                TestResult TestDu
ration(in minutes)
-------------------------------------------------------------------------------------------------------------------
    1 CORE                 VERIFY-LINUX-DISK-SETUP                                             PASS                 3.26
```

File: Testscripts/Linux/VERIFY-LINUX-CONFIGURATION.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    if ('*' in root_passwd or '!' in root_passwd):
+        RunLog.info('root password is deleted in /etc/shadow.')
+        root_password_verify_result = True
+    else:
+        RunLog.error('root password not deleted.%s', passwd_output)
+
+
+def CheckLastConsole(command):
+    global last_console_check_result
+    RunLog.info("Checking for last console as console=ttys0 in  kernel boot line.")
```
Comment (chvalean) [inline Testscripts/Linux/VERIFY-LINUX-CONFIGURATION.py:73]: remove double spacing

File: Testscripts/Linux/VERIFY-LINUX-CONFIGURATION.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            RunLog.info("ClientAliveInterval " + 'is ' + ClientAliveIntervalValue)
+            sshd_config_check_result = True
+        else:
+            print ("CLIENT_ALIVE_INTERVAL_FAIL")
+            RunLog.error("ClientAliveInterval is " + ClientAliveIntervalValue + " is not expected.")
+
+
+def VerifyRootPassword():
+    global root_password_verify_result
+    RunLog.info("Checking if root password is deleted or not...")
```
Comment (chvalean) [inline Testscripts/Linux/VERIFY-LINUX-CONFIGURATION.py:60]: if root password is expected to be deleted, please change the message to below, to make it clear on what is the expected behavior.
"Checking if root password is deleted"

File: Testscripts/Linux/VERIFY-LINUX-DISK-SETUP.py [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+
+def CheckRootDeviceTimeout(command):
+    global root_device_timeout_check_result
+    RunLog.info("Checking root device timeout start...")
+    temp = Run(command)
+    rootDeviceTimeout = 300
+    output = int(temp)
+    if (output == rootDeviceTimeout) :
+        RunLog.info('SDA timeout value is %s', output)
```
Comment (chvalean) [inline Testscripts/Linux/VERIFY-LINUX-DISK-SETUP.py:173]: sda device timeout

**Review by chvalean — APPROVED** [review]
small changes requested but it looks good, thank you!

**Review by juhlee-microsoft — APPROVED** [review]

**Pull request merged**